src/ldp/operations/AuthorizedOperation.ts

```diff
@@ -0,0 +1,12 @@
+import type { Authorization } from '../../authorization/Authorization';
+import type { Operation } from './Operation';
+
+/**
+ * An operation which has been authorized.
+ */
+export interface AuthorizedOperation extends Operation {
```

Comment:
Is there a specific reason why we do not just make all `Operation` instances have the field, but mark it as optional? This pattern makes the inheritance weird.

Reply:
Just to keep it generic (in case in the future we discovered other things we also want to add), but just adding it as an optional field would probably be easier indeed. Would keep calling it an Operation then though, and the `authorization` field existing or not would indicate whether it is authorized.